helevents/models.py

```diff
@@ -9,4 +9,8 @@ def get_display_name(self):
         return '{0} {1}'.format(self.first_name, self.last_name).strip()
 
     def get_default_organization(self):
-        return self.admin_organizations.order_by('created_time').first()
+        org = self.admin_organizations.order_by('created_time').first()
```

Comment:
I think here we should just filter out the replaced organizations. That way we don't accidentally return a replaced organization. This will also make sure that the user has at least one current (non-replaced) organization, so it's not an old user.

Reply:
Just want to make sure, that means those users that assigned to `replaced_organization.admin_users` will return None for this method, which will break backward compatibility unless the admin manually copy the users from old organization to new one?

Reply:
@mingfeng I believe that is the only transition that we should make automatically. While the old objects may remain published by the old organizations, I think we should create an admin command for copying admin_users to the new organizations.

Because we do not want to keep the users scattered around different organizations, I think it is safer to have all the people who may edit data listed under the new organization directly, so we can check permissions without having to check replaced organizations separately.

Reply:
Will do the auto data transition in `django-orghierarchy`, we can leave this PR open until the changes is done there.

Reply:
Perhaps we should do the same for `DataSource`. `DataSource` and `User` are the only two external models referred to in `django-orghierarchy`, and it is better that they are not linked to replaced organizations. That way, also the api keys for data sources automatically map to the new organizations.

Reply:
However, there is the problem that datasource is linkedevents-specific. I'm referring to the datasource `owner` field here. To map the apikeys correctly, that would have to be done in linkedevents :(

Reply:
That's true, it has to be done in linkedevents then.